From: John Rigby <jrigby@freescale.com>
Date: Thu, 24 Aug 2006 12:11:48 -0600
Subject: [PATCH] ppc qconfig

Original qconfig turns off certain drivers if ppc.
This patch turns those drivers back on.

Signed-off-by: John Rigby <jrigby@freescale.com>

---

 qtopia/src/qt/qconfig-qpe.h |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

d7c955ae6cab4c85954f59ba3bc91980079565bf
diff --git a/qtopia/src/qt/qconfig-qpe.h b/qtopia/src/qt/qconfig-qpe.h
index 8b23a1c..6c7525d 100644
--- a/qtopia/src/qt/qconfig-qpe.h
+++ b/qtopia/src/qt/qconfig-qpe.h
@@ -147,8 +147,8 @@ #endif
 
 // Disable these drivers under PowerPC because some of them cause compile/link failures
 #if defined(__powerpc__)
-#define QT_NO_QWS_MACH64
-#define QT_NO_QWS_VNC
-#define QT_NO_QWS_VGA_16
+//#define QT_NO_QWS_MACH64
+//#define QT_NO_QWS_VNC
+//#define QT_NO_QWS_VGA_16
 #define QT_NO_QWS_SVGALIB
 #endif
-- 
1.3.3

